TorchProteinLibrary/FullAtomModel/Angles2Coords/Angles2Coords.py

```diff
@@ -29,60 +29,101 @@ def convertString(string):
 	'''Converts a string to 0-terminated byte tensor'''  
 	return torch.from_numpy(np.fromstring(string+'\0', dtype=np.uint8))
 
+def convert2str(tensor):
```

Comment:
Is this function needed? I don't see it being called anywhere in that file. (Somewhere else, maybe?)